jdk/src/share/classes/java/lang/Character.java

```diff
@@ -2573,6 +2575,17 @@ private UnicodeBlock(String idName, String... aliases) {
                              "ARABIC MATHEMATICAL ALPHABETIC SYMBOLS",
                              "ARABICMATHEMATICALALPHABETICSYMBOLS");
 
+        /**
+         * Constant for the "CJK Unified Ideographs Extension E" Unicode
+         * character block.
+         * @apiNote This field is defined in Java SE 8 Maintenance Release 5.
+         * @since 1.8
```

Comment:
Shouldn't this be `8` instead of `1.8`?

Reply:
IIUC, `@since` tags for JDK 8 all use `1.8`, but happy to revise it if MRs prefer `8` instead.

Reply:
Sounds good.  As I wrote my comment that thought did occur to me.  We should be consistent with the existing tags so `1.8` it is.